Homework_7.py

```diff
@@ -0,0 +1,95 @@
+# Homework 7
+
+import csv  # import module for work with csv
+import os  # import module for ability to delete files
+from Homework_6 import *  # import previous homeworks (5-6)
+
+
+class CountWords:
+    def parse_file(self, list_with_words):
```

Comment:
You don't need list_with_words as argument (you can just create empty list in the beginning of the function if you need it), it is the output of this function. Let's better have the path to the file we need to parse as an argument.